elements/pfe-primary-detail/src/pfe-primary-detail.js

```diff
@@ -0,0 +1,177 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfePrimaryDetail extends PFElement {
+  static get tag() {
+    return "pfe-primary-detail";
+  }
+
+  static get meta() {
+    return {
+      title: "Primary detail",
+      description: ""
+    };
+  }
+
+  get templateUrl() {
+    return "pfe-primary-detail.html";
+  }
+
+  get styleUrl() {
+    return "pfe-primary-detail.scss";
+  }
+
+  // static get events() {
+  //   return {
+  //   };
+  // }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Container;
+  }
+
+  static get properties() {
+    return {};
+  }
+
+  constructor() {
+    super(PfePrimaryDetail, { type: PfePrimaryDetail.PfeType });
+
+    this._handleHideShow = this._handleHideShow.bind(this);
+
+    this._detailsNav = this.shadowRoot.getElementById("details-nav");
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    // If you need to initialize any attributes, do that here
+
+    this._slots = {
+      detailsNav: this.shadowRoot.querySelector('slot[name="details-nav"]').assignedElements(),
```

Comment:
```suggestion
      detailsNav: this.getSlot("details-nav"),
```

PFElement has a `this.getSlot` method you can use and it returns an array of slotted elements.

Reply:
Hi @castastrophe, what version of PFElement is that method in? I am trying to use it in my branch for PFE-NAV 2.0 but getting an error saying it is not a function.

Reply:
PFElement 1.0.0 on line 238 - happy to jump on a call today if I can help!